printers/printerTree.php

```diff
@@ -135,36 +137,50 @@ private function str_contains(string $haystack, string $needle){
         return strpos($haystack, $needle) !== false;
     }
 
-    private function _fillTree($tree, $keys, $item, $parentId) {
+    private function _fillTree($tree, $keys, $item, $parentId, $myNs) {
         if (empty($keys)){ // We've reach the end of the journey. We register the data of $item
             if($item['type'] === 'd') {
                 $tree->self = $item;
             } else {
-                $tree->pages []= $item;
+                if ($myNs == $item['id']) {
+                    $tree->self = $item;
+                } else {
+                    if (!isset($tree->children[$item['id']])) {
+                        if ('d' !== $item['type']) {
+                            $tree->children[$item['id']] = new NspagesTreeNsNode($item['id']);
```

Comment:
I have situation on Centos 7 and php 7.4 that "f" node was before "d" node (which not happend for me on php 8.2), without this line I will lose all subnodes. Also there will be wrong assign: array instead object. With this I create correct entry for f node without children (but children can show after creation).

Reply:
Why using nsKey no ID? because id is not perfect for building tree, For example I have pages: linux:config:start that should show for ns "linux:config" as self. If I use ID, I will have duplicates in tree.